homeassistant/components/modbus/sensor.py

```diff
@@ -112,6 +113,7 @@ async def async_update(self, now: datetime | None = None) -> None:
         if raw_result is None:
```

Comment:
Should `async_update` set `self._cancel_call` to `None` ?

Reply:
Done